Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/CvarAdapter.cpp

```diff
@@ -142,7 +197,7 @@ namespace AZ::DocumentPropertyEditor
         bool BuildEditorForCvar(AdapterBuilder& builder, ConsoleFunctorBase* functor)
         {
             // Check all possible type IDs for our supported types and try to create a property editor
-            return TryBuildNumericEditor<int8_t, int16_t, int32_t, int64_t, uint8_t, uint16_t, uint32_t, uint64_t, float, double>(
+            return TryBuildNumericEditor<AZ::s8, AZ::u8, AZ::s16, AZ::u16, AZ::s32, AZ::u32, AZ::s64, AZ::u64, long, unsigned long, float, double>(
```

Comment:
question: Is it preferred to use the AZ aliases? I thought using the standard names was actually preferred these days? (those in `<cstdint>`)

Reply:
There is an issue with using `int64_t` and `uint64_t` having a different type on Linux(`long`, `unsigned long) vs Windows(`long long`, `unsigned long long`).
Therefore in order to support all the existing numeric types that Cry Variables uses with the old CVAR system, we need to specify the "other" `long` or `long long` type on the platform to make sure all types are accounted for.
As the AZ s64 and u64 aliases are always `long long`, `unsigned long long` on all of O3DE platforms, then only `long`, `unsigned long` needs to be accounted for.